docs/experiment/ppl/functions/condition.rst

```diff
@@ -67,3 +67,78 @@ Example, the account 13 doesn't have email field::
     | 13               | null    |
     +------------------+---------+
 
+IFNULL
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: ifnull(field1, field2) return field2 if field1 is null.
+
+Argument type: all the supported data type, (NOTE : if two parametershas different type, return result is correct, but return type could be wrong)
+
+Return type: any
```

Comment:
Fo argument type, it should be "if two parameters have different type, you will fail semantic check"?

For return type, to avoid confusion, could you elaborate a little more than "any"?

Same for other function docs in PPL and SQL added.